website/api/views.py

```
@@ -245,9 +300,42 @@ def create(self, request, *args, **kwargs):
+
+                    try:
+                        original_cve_id = issue.cve_id
+                        normalize_and_populate_cve_score(issue)
+                        update_fields = []
+                        if issue.cve_id != original_cve_id:
+                            update_fields.append("cve_id")
+                        if issue.cve_score is not None:
+                            update_fields.append("cve_score")
+                        if update_fields:
+                            issue.save(update_fields=update_fields)
+                    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)